contrib/babelfishpg_tds/error_mapping.txt

```diff
@@ -176,3 +176,4 @@ XX000 ERRCODE_INTERNAL_ERROR	"The table-valued parameter \"%s\" must be declared
 42P01 ERRCODE_UNDEFINED_TABLE	"table \"%s\" does not exist"		SQL_ERROR_3701	16
 42P13 ERRCODE_INVALID_FUNCTION_DEFINITION   "PL/tsql functions cannot return type %s"   SQL_ERROR_2733 16
 0A000 ERRCODE_FEATURE_NOT_SUPPORTED	"Column name or number of supplied values does not match table definition."	SQL_ERROR_213	16
+42501 ERRCODE_INSUFFICIENT_PRIVILEGE	"Only members of the %s role can execute this stored procedure."	SQL_ERROR_15003	16
```

Comment:
If we are mapping a new error from PG to SQL server, we should first figure out the error behaviour to make sure we are compliant with that. Please talk to Dipesh to find more details.